api/namex/utils/queue_util.py

```diff
@@ -0,0 +1,50 @@
+import json
+import time
+import uuid
+from datetime import datetime, timezone
+
+from config import Config
+from namex.services import queue
+
+
+def create_cloud_event_msg(msg_id, msg_type, source, time, identifier, json_data_body):  # pylint: disable=too-many-arguments # noqa E501
```

Comment:
why not just import https://github.com/bcgov/namex/blob/main/services/common/src/queue_common/messages.py

Reply:
Ok. Done.